src/js/components/Form/stories/ValidateOnBlur.js

```diff
@@ -37,12 +37,17 @@ export const ValidateOnBlur = () => (
               return undefined;
             },
           ]}
-        />
-
+        >
+          <TextInput aria-required name="name" />
+        </FormField>
         <FormField label="Email" name="email" required>
-          <TextInput name="email" aria-label="email" type="email" />
+          <TextInput
+            aria-required
+            name="email"
+            aria-label="email"
```

Comment:
Instead of `aria-label` use `htmlFor` and `id`

Reply:
done